docs/olm-bundle-management.md

```
@@ -112,20 +112,21 @@ make bundle-push BUNDLE_IMG=quay.io/myorg/lightspeed-operator-bundle:v0.1.0
-  "name": "lightspeed-operator",
-  "image": "quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/lightspeed-operator@sha256:65b288d147503bd66808eecf69deee8e71ec890752a70f83a9314061fe0d4420",
-  "revision": "e5e1454f3fa8b19293200868684abcaf18f38097"
+  "name": "lightspeed-service-api",
+  "image": "quay.io/.../lightspeed-service@sha256:...",
+  "revision": "e5e1454f3fa8b19293200868684abcaf18f38097",
+  "operator_arg": "service-image"
 }
 ```
 
 **Workflow:**
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)